src/components/checkbox/__tests__/Checkbox.Test.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { cleanup, render, screen } from '@testing-library/react';
+import { userEvent } from '@storybook/testing-library';
+import { expect } from '@storybook/jest';
+
+import Checkbox from '../index';
+
+const renderWrapper = ({
+  name = 'Control',
+  onChange = jest.fn(),
+  label = 'Checkbox label',
+  isDisabled = false,
+  isChecked = false,
+} = {}) =>
+  render(
+    <Checkbox
+      name={name}
+      onChange={onChange}
+      renderLabel={label}
+      isDisabled={isDisabled}
+      isChecked={isChecked}
+    />
+  );
+
+describe('<Checkbox>', () => {
+  beforeEach(cleanup);
+
+  it('renders checkbox with label', () => {
+    renderWrapper();
+    const checkbox = screen.getByRole('checkbox', { name: 'Checkbox label' });
+
+    expect(checkbox).toBeInTheDocument();
```

Comment:
I think this is not needed because the previous command would already fail if it's not in the document, no?

Reply:
Not sure about what you mean here. I think, I need it for check if the checkbox exists.

Reply:
I am pretty sure that `screen.getByRole('checkbox', { name: 'Checkbox label' })` fails if the checkbox does not exist. So it's enough to just do `expect(screen.getByRole('checkbox', { name: 'Checkbox label' }))` and remove line 32

Reply:
Mmm but the expect must have a call. Otherwise, it will through an error.

Reply:
Let's leave it as is. While it is true that the `getByRole` would raise if the element does not exist other solutions (no expect call, no expect at all) are against jest conventions and our lint rules.